integration-tests/js-compute/fixtures/extend-from-builtins/extend-from-builtins.ts

```diff
@@ -1,4 +1,6 @@
 /// <reference types="@fastly/js-compute" />
+declare const CompressionStream: any;
+declare const ObjectStore: any;
```

Comment:
Thanks for fixing this! It might be worth pulling this out into its own PR just to cut down on the noise during testing.